WooCommerce/Classes/ViewRelated/Products/Edit Product/Inventory Settings/ProductInventorySettingsViewController.swift

```diff
@@ -0,0 +1,295 @@
+import UIKit
+import Yosemite
+
+final class ProductInventorySettingsViewController: UIViewController {
+
+    @IBOutlet private weak var tableView: UITableView!
+
+    // Editable data - shared.
+    //
+    private var sku: String?
+    private var manageStockEnabled: Bool
+    private var soldIndividually: Bool
+
+    // Editable data - manage stock enabled.
+    private var stockQuantity: Int?
+    private var backordersSetting: ProductBackordersSetting?
+
+    // Editable data - manage stock disabled.
+    private var stockStatus: ProductStockStatus?
+
+    /// Table Sections to be rendered
+    ///
+    private var sections: [Section] = []
+
+    init(product: Product) {
+        self.sku = product.sku
+        self.manageStockEnabled = product.manageStock
+        self.soldIndividually = product.soldIndividually
+
+        self.stockQuantity = product.stockQuantity
+        self.backordersSetting = product.backordersSetting
+        self.stockStatus = product.productStockStatus
+
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureNavigationBar()
+        configureMainView()
+        configureTableView()
+        reloadSections()
+    }
+}
+
+// MARK: - View Updates
+//
+private extension ProductInventorySettingsViewController {
+    func reloadSections() {
+        let stockSection: Section
+        if manageStockEnabled {
+            stockSection = Section(rows: [.manageStock, .stockQuantity, .backorders])
+        } else {
+            stockSection = Section(rows: [.manageStock, .stockStatus])
+        }
```

Comment:
This could be condensed to:

```
var stockSection = Section(rows: [.manageStock, .stockStatus])
if (manageStockEnabled) {
    stockSection.append(.backorders)
}
```

Reply:
I slightly prefer using `let` to `var` so that it can only be set in the same conditional context, otherwise it could be set to a different value later in the function